src/System.IO.FileSystem/tests/Directory/Move.cs

```diff
@@ -144,14 +144,26 @@ public void Path_Longer_Than_MaxPath_Throws_Exception()
 
         [Fact]
         [PlatformSpecific(PlatformID.Windows)]
-        public void Path_With_Longer_Than_MaxDirectory_Throws_Exception()
+        public void Path_With_Longer_Than_MaxDirectory_Succeeds()
         {
             string testDir = GetTestFilePath();
             Directory.CreateDirectory(testDir);
+            Assert.True(Directory.Exists(testDir), "test directory should exist");
             Assert.All((IOInputs.GetPathsLongerThanMaxDirectory()), (path) =>
             {
-                Assert.Throws<PathTooLongException>(() => Move(testDir, path));
-                Assert.Throws<PathTooLongException>(() => Move(path, testDir));
+                string baseDestinationPath = Path.GetDirectoryName(path);
+                if (!Directory.Exists(baseDestinationPath))
```

Comment:
For troubleshooting when tests fail. If the prereq's aren't there you can rat hole into collateral fallout.